Issutracker : #i53288#
CWS         : N/A
Author:     : <maho@openoffice.org> (JCA)
Description : remove circular dependency

in this case,
core01 is depend on core02-core08, and core0[2-9] and core10
depends on core01. for FreeBSD this is regarded as circular dependency
and pkg_add command is forked infinite times and eats up systems' resource.
namely when core01 tries to install core02, core02 also tries to
install core01.

Index: instsetoo_native/inc_openoffice/unix/packagelist.txt
===================================================================
RCS file: /cvs/installation/instsetoo_native/inc_openoffice/unix/packagelist.txt,v
retrieving revision 1.16
diff -u -r1.16 packagelist.txt
--- instsetoo_native/inc_openoffice/unix/packagelist.txt	30 Aug 2005 08:52:34 -0000	1.16
+++ instsetoo_native/inc_openoffice/unix/packagelist.txt	7 Sep 2005 10:20:37 -0000
@@ -36,7 +36,7 @@
 solarisrequires = "openofficeorg-core02,openofficeorg-core03,openofficeorg-core04,openofficeorg-core05,openofficeorg-core06,openofficeorg-core07,openofficeorg-core08"
 packagename = "openoffice.org-core01"
 linuxreplaces = "openofficeorg-mailcap,openofficeorg-core,openofficeorg-core01"
-requires = "openoffice.org-core02,openoffice.org-core03,openoffice.org-core04,openoffice.org-core05,openoffice.org-core06,openoffice.org-core07,openoffice.org-core08"
+requires = "openoffice.org-core02,openoffice.org-core03,openoffice.org-core04,openoffice.org-core05,openoffice.org-core06,openoffice.org-core07,openoffice.org-core08,openoffice.org-core09,openoffice.org-core10"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -265,7 +265,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core02"
 linuxreplaces = "openofficeorg-core02"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -278,7 +277,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core03"
 linuxreplaces = "openofficeorg-core03"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -291,7 +289,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core04"
 linuxreplaces = "openofficeorg-core04"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -304,7 +301,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core05"
 linuxreplaces = "openofficeorg-core05"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -317,7 +313,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core06"
 linuxreplaces = "openofficeorg-core06"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -330,7 +325,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core07"
 linuxreplaces = "openofficeorg-core07"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -343,7 +337,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core08"
 linuxreplaces = "openofficeorg-core08"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -356,7 +349,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core09"
 linuxreplaces = "openofficeorg-core09"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
@@ -370,7 +362,6 @@
 solarisrequires = "openofficeorg-core01"
 packagename = "openoffice.org-core10"
 linuxreplaces = "openofficeorg-core10"
-requires = "openoffice.org-core01"
 copyright = "1999-2005 by OpenOffice.org"
 vendor = "OpenOffice.org"
 description = "Office core module for %PRODUCTNAME %PRODUCTVERSION"
